.travis.yml

```diff
@@ -100,16 +100,17 @@ script:
   - git submodule update --quiet --init --recursive
   - echo 'Building POSIX Firmware..' && make posix_sitl_default
   - echo 'Running Tests..' && make posix_sitl_default test
+  - build_configs="px4fmu-v1_default px4fmu-v2_default px4fmu-v2_lpe \
+        mindpx4-v2_default uavcan_firmware \
+        px4fmu-v4_default px4-stm32f4discovery_default"
   - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then
          cd ${TRAVIS_BUILD_DIR}
       && make check_format
       && arm-none-eabi-gcc --version
-      && echo 'Building NuttX px4fmu-v1 Firmware..' && make px4fmu-v1_default
-      && echo 'Building NuttX px4fmu-v2 Firmware..' && make px4fmu-v2_default
-      && echo 'Building NuttX mindpx-v2 Firmware..' && make mindpx-v2_default
-      && echo 'Building UAVCAN node firmware..' && make uavcan_firmware
-      && echo 'Building NuttX px4fmu-v4 Firmware..' && make px4fmu-v4_default
-      && echo 'Building NuttX px4-stm32f4discovery Firmware..' && make px4-stm32f4discovery_default
+      && build_configs="px4fmu-v1_default px4fmu-v2_default px4fmu-v2_lpe \
```

Comment:
Let's just pull this into the Makefile under `make check`. We just need to be careful to build uavcan_firmware and px4fmu-v4_default last and in that order.
Developers can then just run check locally to match what the builds do.